appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -187,4 +193,18 @@ export class CaseDetailsPage extends Page {
 				);
 		});
 	}
+
+	verifyChangeStartDate() {
+		const date = new Date(); // Note: Months are 0-indexed, so 10 is November
+		const formattedDate = new Intl.DateTimeFormat('en-GB', {
```

Comment:
Once Jacob has merged in his PR which pulls this code into the `DateTimeSection` file, we can just reference that here